### PR TITLE
removed if statement for electron from nightly workflow

### PR DESCRIPTION
**Pull request by wojteknowacki**
closes #...

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `data-test-id` are added for new elements
4. [ ] The changes are tested in Chrome/Firefox/Safari browsers and in light/dark mode
5. [ ] Your code works with the latest stable version of the core
6. [ ] I added changesets file (instructions in [contribution guide](https://github.com/saleor/saleor-dashboard/blob/main/.github/CONTRIBUTING.md)

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] app
3. [ ] attribute
4. [ ] category
5. [ ] collection
6. [ ] customer
7. [ ] giftCard
8. [ ] homePage
9. [ ] login
10. [ ] menuNavigation
11. [ ] navigation
12. [ ] orders
13. [ ] pages
14. [ ] payments
15. [ ] permissions
16. [ ] plugins
17. [ ] productType
18. [ ] products
19. [ ] sales
20. [ ] shipping
21. [ ] translations
22. [ ] variants
23. [ ] vouchers

CONTAINERS=2

### CODE REVIEW

**Review by andrzejewsky — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

File: .github/workflows/tests-nightly.yml [hunk 1]
```diff
@@ -167,7 +167,7 @@ jobs:
 
       - name: Cypress run electron
         id: cypress-electron
-        if: ${{ github.event.inputs.tests != 'Critical' && github.event_name != 'repository_dispatch' && contains(fromJSON('["chrome", "all"]'), steps.get-browsers.outputs.result) && ! cancelled() }}
+        if: ${{ github.event.inputs.tests != 'Critical' && github.event_name != 'repository_dispatch' && ! cancelled() }}
```
Comment (karola312) [inline .github/workflows/tests-nightly.yml:170]: Won't be better if we just change contains(fromJSON('["chrome", "all"]') to contains(fromJSON('["electron", "all"]')

Comment (karola312) [reply .github/workflows/tests-nightly.yml:170]: If we leave it like this, when you manually trigger tests for eg. on firefox, tests on electron and on firefox will be executed

Comment (andrzejewsky) [reply .github/workflows/tests-nightly.yml:170]: seems like there is input on top, that allows you to use the only browsers listed there, so there is no need to have additional validation here

**Review by karola312 — DISMISSED** [review]

**Review by andrzejewsky — APPROVED** [review]

File: .github/workflows/tests-nightly.yml
Comment (NyanKiyoshi) [inline .github/workflows/tests-nightly.yml:1]: This line is still referencing the `chrome` input: https://github.com/saleor/saleor-dashboard/blob/41525d3345602922800a4266ca6841f6c7a22e34/.github/workflows/tests-nightly.yml#L214

**Review by NyanKiyoshi — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**